docs/app-center-preparing-assets.md

```diff
@@ -0,0 +1,124 @@
+---
+id: app-center-preparing-assets
+title: App Center
+sidebar_label: Preparing assets
+---
+
+
+##### Preparing assets for publishing your Aragon app
+
+Before you publish your Aragon app in the App Center, you need to prepare a few types of assets to display it correctly, and differentiate from other apps.
+
+ The main assets are: 
+- App Menu Icon (SVG)
+- Large app icon (SVG or PNG)
+- App Badge Icon (SVG or PNG)
+- Screenshots (PNG)
+- Short description (text only)
+- Long description (Markdown or text)
+
+<br>
+In the next sections we will go through these one by one, as well as link to some templates so it’s easy for you to prepare everything that you need.
+
+<br>
+
+
+## Icons
+
+### Menu icon
+
+This icon is used on the left hand app navigation panel of Aragon.
+<br>
+<br>
+<span>![*](/docs/assets/check.svg) **Size** of the icon including padding is **22x22 px**. The padding should be around 3px or more on each side, depending on the icon shape</span>
+
+<span>![*](/docs/assets/check.svg) **Color** of the icon should be **#888888** so that we can automatically colorize the icon for you correctly in the Aragon client for various icon states</span>
```

Comment:
```suggestion
<span>![*](/docs/assets/check.svg) **Color** of the icon should be **#888888** so that the Aragon client can automatically colorize the icon in different app states</span>
```